gamification/models.py

```diff
@@ -186,3 +187,13 @@ def add_badges(self, activities):
         badges = self.find_badges_for_activity(activities)
         for badge in badges:
             self.badges.add(badge)
+
+
+def meta_review_add_activity(self):
+    """
+    Add a meta-review activity to the participant
+    """
+    activity = Reaction.object.create(
```

Comment:
This part does not make sense to me. I don't understand what you are doing with `Reaction` here. Once you know if a participant has finished the meta-review process, the rest staff has nothing to do with the meta-review app.

Reply:
What you said at https://github.com/coala/community/pull/190#issuecomment-423495158 looks good. However, you are not handling `update the activity` correctly.

Reply:
> This part does not make sense to me. I don't understand what you are doing with `Reaction` here. Once you know if a participant has finished the meta-review process, the rest staff has nothing to do with the meta-review app.

In ```meta_review_add_activity ```, I'm using the details from```Reaction``` class of ```meta-review``` app to add the ```Completed Meta-Review``` activity (In the same way as that of ```add_activity``` in the ```gamification```.



> What you said at [#190 (comment)](https://github.com/coala/community/pull/190#issuecomment-423495158) looks good. However, you are not handling `update the activity` correctly here.
> 
> Moreover, this should not be a standalone function. At least make it a method of `Participant` class in `gamification/models.py`. Maybe we don't even need it - you can simply put the logic in `update_participants_data_with_meta_review` function. After you implement the correct logic, we can start talking about coding style.

I added this as a function because of the pattern followed by ```add_activity``` of ```gamification```.

If you feel that I should remove this function as a whole and implement the logic in ```update_participants_data_with_meta_review```, I can do it.

Reply:
> I'm using the details fromReaction class of meta-review app to add the Completed Meta-Review activity

An activity should be an instance of `Activity` class. When you do `Reaction.object.create` you are creating an instance of `Reaction`, which does not make sense (and `Reaction` does not have `activity` field btw).

> I added this as a function because of the pattern followed by add_activity of gamification.

Okay, I didn't notice that point ;) Then don't put it in `models.py`. But we can put this thing aside and make the logic go correctly first.